### PR TITLE
DOCS: Update Consul Connect to Consul service mesh

### PR DESCRIPTION
**Pull request by angrycub**
Updating references to Consul Connect to be Consul service mesh for consistency.

### CODE REVIEW

File: website/content/docs/integrations/consul-connect.mdx [hunk 1]
```diff
@@ -1,70 +1,70 @@
...
-[Consul Connect](https://www.consul.io/docs/connect) provides
+[Consul service mesh](https://www.consul.io/docs/connect) provides
 service-to-service connection authorization and encryption using mutual
 Transport Layer Security (TLS). Applications can use sidecar proxies in a
 service mesh configuration to automatically establish TLS connections for
-inbound and outbound connections without being aware of Connect at all.
+inbound and outbound connections without being aware of service mesh at all.
 
-# Nomad with Consul Connect Integration
+# Nomad with Consul service mesh integration
```
Comment (schmichael) [inline website/content/docs/integrations/consul-connect.mdx:23]: Shouldn't it still be Title Cased in headings as in `# Consul Service Mesh` above?

Comment (angrycub) [reply website/content/docs/integrations/consul-connect.mdx:23]: Will make title case. In the Learn style guide, there is only one H1 per page (the proper title) and all other headings use  sentence case.  We might not be there on docs yet.
```suggestion
# Nomad with Consul Service Mesh Integration
```

File: website/content/docs/integrations/consul-connect.mdx [hunk 2]
```diff
@@ -100,16 +100,16 @@ For JSON configurations:
 
 Nomad must schedule onto a routable interface in order for the proxies to
 connect to each other. The following steps show how to start a Nomad dev agent
-configured for Connect.
+configured for Consul service mesh.
 
 ```shell-session
-$ sudo nomad agent -dev-connect
+$ sudo nomad agent -dev-service mesh
```
Comment (angrycub) [inline website/content/docs/integrations/consul-connect.mdx:106]:
```suggestion
$ sudo nomad agent -dev-connect
```

File: website/content/docs/integrations/consul-connect.mdx [hunk 3]
```diff
@@ -140,11 +140,11 @@ net.bridge.bridge-nf-call-ip6tables = 1
 net.bridge.bridge-nf-call-iptables = 1
 ```
 
-## Run the Connect-enabled Services
+## Run the service mesh-enabled services
```
Comment (angrycub) [inline website/content/docs/integrations/consul-connect.mdx:143]:
```suggestion
## Run the Service Mesh-enabled Services
```

File: website/content/docs/integrations/consul-connect.mdx [hunk 4]
```diff
@@ -287,7 +287,7 @@ This allows you to connect to the web frontend in a browser by visiting
 
 [![Count Dashboard][count-dashboard]][count-dashboard]
 
-The web frontend connects to the API service via Consul Connect:
+The web frontend connects to the API service via Consul service mesh:
```
Comment (angrycub) [inline website/content/docs/integrations/consul-connect.mdx:290]:
```suggestion
The web frontend connects to the API service via Consul service mesh.
```

File: website/content/docs/integrations/consul-connect.mdx [hunk 5]
```diff
@@ -1,70 +1,70 @@
...
 
 ### Consul
 
-Connect integration with Nomad requires [Consul 1.6 or
+The Consul service mesh integration with Nomad requires [Consul 1.6 or
 later.](https://releases.hashicorp.com/consul/1.6.0/) The Consul agent can be
 run in dev mode with the following command:
 
-**Note**: Nomad's Connect integration requires Consul in your `$PATH`
+**Note**: Nomad's service mesh integration requires Consul in your `$PATH`
```
Comment (angrycub) [inline website/content/docs/integrations/consul-connect.mdx:61]:
```suggestion
**Note**: Nomad's Consul service mesh integration requires Consul in your `$PATH`
```

File: website/content/docs/integrations/consul-connect.mdx [hunk 6]
```diff
@@ -1,70 +1,70 @@
...
 
-# Nomad Consul Connect Example
+# Nomad Consul service mesh example
 
 The following section walks through an example to enable secure communication
 between a web dashboard and a backend counting service. The web dashboard and
 the counting service are managed by Nomad. Nomad additionally configures Envoy
 proxies to run along side these applications. The dashboard is configured to
-connect to the counting service via localhost on port 9001. The proxy is managed
+service mesh to the counting service via localhost on port 9001. The proxy is managed
```
Comment (angrycub) [inline website/content/docs/integrations/consul-connect.mdx:50]:
```suggestion
connect to the counting service via localhost on port 9001. The proxy is managed
```

File: website/content/docs/integrations/consul-connect.mdx [hunk 7]
```diff
@@ -1,70 +1,70 @@
...
 or down or gets rescheduled by Nomad.
 
-For using the Consul Connect integration with Consul ACLs enabled, see the
-[Secure Nomad Jobs with Consul Connect](https://learn.hashicorp.com/tutorials/nomad/consul-service-mesh)
+For using the Consul service mesh integration with Consul ACLs enabled, see the
+[Secure Nomad Jobs with Consul Service Mesh](https://learn.hashicorp.com/tutorials/nomad/consul-service-mesh)
 guide.
 
-# Nomad Consul Connect Example
+# Nomad Consul service mesh example
```
Comment (angrycub) [inline website/content/docs/integrations/consul-connect.mdx:44]:
```suggestion
# Nomad Consul Service Mesh Example
```

File: website/content/docs/integrations/consul-connect.mdx [hunk 8]
```diff
@@ -1,70 +1,70 @@
...
+~> **Note:** Nomad's service mesh integration requires Linux network namespaces.
+Consul service mesh will not run on Windows or macOS.
 
-[Consul Connect](https://www.consul.io/docs/connect) provides
+[Consul service mesh](https://www.consul.io/docs/connect) provides
 service-to-service connection authorization and encryption using mutual
 Transport Layer Security (TLS). Applications can use sidecar proxies in a
 service mesh configuration to automatically establish TLS connections for
-inbound and outbound connections without being aware of Connect at all.
+inbound and outbound connections without being aware of service mesh at all.
```
Comment (angrycub) [inline website/content/docs/integrations/consul-connect.mdx:21]:
```suggestion
inbound and outbound connections without being aware of the service mesh at all.
```

**Pull request merged**